codeceptConfig.js

```diff
@@ -52,14 +54,22 @@ module.exports = {
     templatesAPI: './tests/ia/pages/api/templatesAPI.js',
   },
   getChunks: (files) => {
-    const dependentTests = files.filter((value) => /PMMSettings|ia|backup/.test(value));
-    const dbaasTests = files.filter((value) => /DbaaS/.test(value));
-    const otherTests = files.filter((val) => !dependentTests.includes(val) && !dbaasTests.includes(val));
+    const take = (regexp) => {
+      const next = files.filter((value) => regexp.test(value));
+      // eslint-disable-next-line no-param-reassign
+      files = files.filter((v) => !next.includes(v));
+      return next;
+    };
+
+    const _ = require('lodash');
+    const splitIntoParts = (items, parts) => _.chunk(items, items.length / parts);
 
     return [
-      dependentTests,
-      otherTests,
-      dbaasTests,
+      ...splitIntoParts(take(/\/ia\//), 2),
```

Comment:
easy way to split tests, for cases where is make sense (e.g. tests can run in parallel)

Reply:
Wouldn't this conflict with two IA tests try to change settings?